### PR TITLE
Assignment Submit 03

### PR DESCRIPTION
**Pull request by ymmy1**

### CODE REVIEW

File: Lesson04/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
```
Comment (jimmynono) [inline Lesson04/index.html]: I know it's not part of this assignment, but make sure and put in the viewport meta tag on ALL HTML files moving forward. It's good habit.

File: Lesson04/index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+		<title>Document</title>
+	</head>
+	<body>
+		<button onclick="numberMultiplier()">Click me!</button>
+		<script src="./script.js" charset="utf-8"></script>
```
Comment (jimmynono) [inline Lesson04/index.html]: love the 
```
./
```

File: Lesson04/script.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+function numberMultiplier(){
+	let x = prompt('Enter first number');
+	let y = prompt('Enter second number');
+	let result = 0;
+
+	result = (x * x) * (y * y);
+
+	alert(x+ "*"+  x+ "*"+ "("+ y+ "*"+ y+ ")"+ "="+ result);
```
Comment (jimmynono) [inline Lesson04/script.js]: make sure and format your code right. spaces around all operators

```
alert(x + "*" + x + "*" + "(" + y + "*"+ y+ ")" + " = "+ result); 
```

File: Lesson03/exercise-one.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const workingWeek = ['Monday', 'Tuesday', 'Saturday', 'Wednesday', 'Thursday'];
+
```
Comment (ClovisI) [inline Lesson03/exercise-one.js]: The assignment wanted a console.log for each item.  Here you would only need to list the array:
workingWeek

File: Lesson03/exercise-one.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+workingWeek.push('Friday.');
+workingWeek.splice(2,1);
```
Comment (ClovisI) [inline Lesson03/exercise-one.js]: Just add after push on next line:
workingWeek

File: Lesson03/exercise-one.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+const workingWeek = ['Monday', 'Tuesday', 'Saturday', 'Wednesday', 'Thursday'];
+
+workingWeek.push('Friday.');
```
Comment (ClovisI) [inline Lesson03/exercise-one.js]: console.log(workingWeek);

**Comment by ymmy1** [comment]
I couldn't find anything about this meta tag in our lessons so I had to google.
The tag I added to each html is meta name="viewport" content="width=device-width, initial-scale=1.0" 

Please explain me what is this tag for, and did I do it right?

Thanks.


>Sat,  4 May 2019 -6:49:19 -0700 from James Norton <notifications@github.com>:
>
>@jimmynono commented on this pull request.
>----------------------------------------------------------------------
>In  Lesson04/index.html :
>> @@ -0,0 +1,11 @@
>+<!DOCTYPE html>
>+<html lang="en">
>+	<head>
>+		<meta charset="UTF-8">
>
>I know it's not part of this assignment, but make sure and put in the viewport meta tag on ALL HTML files moving forward. It's good habit.
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .

С уважением,
Oleg Nosyrev
chem-pion@mail.ru

**Comment by jimmynono** [comment]
> I couldn't find anything about this meta tag in our lessons so I had to google. The tag I added to each html is meta name="viewport" content="width=device-width, initial-scale=1.0"  Please explain me what is this tag for, and did I do it right? 

We reference it in the slides for week one. There is a great article at MDN on what it does. You'll want to put in on every website you make moving forward. 

https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag

**Pull request closed without merging**